### DISCUSSION TITLE
When an item is renamed "invisible" it turns invisible.

### DISCUSSION THREAD

**Discussion by Flashtzu** [BedrockTweaks/Files#357] [Ideas]
Hi, I am new here, and relatively new to minecraft. I have a skin I like and was hoping to find a pack that made it so you could rename items and then they would no longer show up. I think I would mainly use this for armor but also maybe glass in some places and some items frames. Thank you for your time.

**Comment by SquatchHunter** [comment]
whilst possible, probably out of scope for the project as it stands atm
an interesting idea non the less and something we will come back to revisit

**Reply by Flashtzu** [reply]
Thank you for the reply. I look forward to more stuff from this community so far everything I have tried has been great.
